apps/hobbygroups/models.py

```diff
@@ -8,6 +8,7 @@ class Hobby(models.Model):
     description = models.TextField(max_length=250)
     image = models.ImageField(upload_to=settings.IMAGES_PATH)
     read_more_link = models.URLField(blank=True)
+    hobby_priority = models.IntegerField(default=0)
```

Comment:
`Hobby.hobby_priority` is a bit redundant. Could simply name it "priority" rather than hobby_priority, since we know we're on a Hobby object already.